data/sebashmh.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Sebastian Henao Madrigal",
+  "type": "personal",
+  "bio": "Computer Scientist, Engineering Manager, FullStack Engineer, Tech entrepreneur, Travel enthusiast, Photographer & Coffee Nerd",
+  "tags": [
+    "Open Source",
+    "JavaScript",
+    "Python",
+    "DevOps",
+    "Kubernetes",
+    "Docker",
+    "Node.js",
+    "AWS"
+  ],
+    "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/madriantor" },
+    { "icon": "FaGithub", "url": "https://github.com/sebashmh" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/sebastianhenaomadrigal/" },
+    { "icon": "FaInstagram", "url": "https://vero.co/sebasmadrigal" },
+    { "icon": "FaLink", "url": "https://sebas.io" },
+  ],
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/madrinator",
+      "icon": "FaTwitter"
+    },
+        {
+      "name": "GitHub",
+      "url": "https://github.com/sebashmh",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/sebastianhenaomadrigal/",
+      "icon": "FaLinkedin"
+    },
```

Comment:
Hey, take out the comma after the LinkedIn object. That's what made the JSON check fail.

Reply:
The comma was removed. @MahirMahdi thanks for the feedback.